src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -317,7 +316,7 @@ class ZStackAdapter extends Adapter {
         networkAddress: number,
         clusterId: Zdo.ClusterId,
         payload: Buffer,
-        disableResponse: true,
+        disableResponse: boolean,
```

Comment:
Switching to this "internal" function breaks proper typing, since that means it allows awaiting for response even though it shouldn't (doesn't have one for given cluster). The 2 overloads are not used anymore for the "internal" function.